src/haz3lcore/zipper/action/Action.re

```diff
@@ -8,10 +8,15 @@ type move =
   | Local(planar)
   | Goal(Measured.Point.t);
 
+[@deriving (show({with_path: false}), sexp, yojson)]
+type jump_target =
+  | TileId(Id.t)
+  | BindingSiteOfIndicatedVar;
+
 [@deriving (show({with_path: false}), sexp, yojson)]
 type t =
   | Move(move)
-  | JumpToId(Id.t)
+  | Jump(jump_target)
   | Select(move)
```

Comment:
Now that I've been doing some similar in Select, I'm somewhat tempted to consolidate these actions under Move.... make these Move(Id(id)), Move(ToDefinition), and make existing Move(whatever) to Move(Relative(whatever))... not quite sure how I feel about this change though. @cyrus- thoughts?

Reply:
In the long term, it would make sense to think of a semantic taxonomy for the actions. Something similar to the context menu breakdown; with variants for `Navigation`, `Refactoring`, `Selection`, etc. I updated the rest of the pr and can collapse Jump/Move for now if we want though.

Reply:
sounds reasonable. just leave it for now, a separate reorg pass is probably in order in a little while.